allegro.h

```diff
@@ -580,11 +580,16 @@ typedef class Serial_read_buffer : public Serial_buffer {
     double get_double() { assert(!(((intptr_t) ptr) & 7));
                           double d = *((double *) ptr); ptr += sizeof(double); 
                           return d; }
-    const char *get_string() { char *s = ptr; char *fence = buffer + len;
-                         assert(ptr < fence);
-                         while (*ptr++) assert(ptr < fence);
-                         get_pad();
-                         return s; }
+    const char *get_string() {
+        char *s = ptr;
+#ifndef NDEBUG
+        const char *fence = buffer + len;
+        assert(ptr < fence);
+        while (*ptr++) { assert(ptr < fence); }
```

Comment:
I see {...} expanded to multiple lines below but not here. Either the original or multiple lines would be preferable to this.

Reply:
Good catch, I missed that one. Fixed 👍